src/SAWScript/Crucible/LLVM/Builtins.hs

```diff
@@ -304,6 +304,22 @@ crucible_llvm_array_size_profile assume bic opts (Some lm) nm lemmas setup = do
     profiles <- io $ readIORef cell
     pure . fmap (\(fnm, prof) -> (Text.unpack fnm, prof)) $ Map.toList profiles
 
+llvmURI :: String -> URI
+llvmURI symbol_name =
+  fromMaybe (error $ unwords ["mkLLVMName", "Could not create LLVM symbol name", symbol_name]) $
```

Comment:
Is this call to `error`believed to be unreachable? Assuming so, it probably ought to be a call to `panic`; are we using `panic` within saw-script yet?

Reply:
It looks like `panic` hasn't migrated up to SAWScript yet.  I think this error should be impossible, but I'll have to figure out if there are conditions under which `mkPathPiece` might fail.